libs/content/workspace-ffi/SwiftWorkspace/Sources/Workspace/iOSMTK.swift

```diff
@@ -106,26 +122,30 @@ public class iOSMTKTextInputWrapper: UIView, UITextInput, UIDropInteractionDeleg
 
         addSubview(floatingCursor)
     }
-    
-    @objc func handleTrackpadScroll(_ sender: UIPanGestureRecognizer? = nil) {
+
+    @objc func handlePan(_ sender: UIPanGestureRecognizer? = nil) {
         mtkView.handleTrackpadScroll(sender)
     }
```

Comment:
While intended for the trackpad, this is a pan handler, so this name makes more sense to me. Vaguely more google-able. I would rename the `mtkView` function too, but actually there's already a `handlePan` there. This one's used for markdown and that one for drawing. Expect more renames like this after I clean that up.